### PR TITLE
CLI: enable comma delimited list of post ids to classifai

### PR DESCRIPTION
**Pull request by adamsilverstein**
### Description of the Change

Explode the comma delimited list of post ids passed to the command.
Adjust the command docs to make it clear what should be passed.

### Verification Process

type a comma delimited list of posts to classifai, eg:

`wp classifai post 3602,3604,3606`

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/classifai-for-wordpress/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@adamsilverstein is there a parallel update to the readme that should explicitly note how this works / provide an example?

**Comment by adamsilverstein** [comment]
@jeffpaul the PR updates the docs you get when typing the CLI command, not sure if this is documented elsewhere, I'll look

**Comment by jeffpaul** [comment]
@adamsilverstein can you please update the readme as well to denote the update to the command?

**Comment by adamsilverstein** [comment]
Added  docs to the readme (as well as some editor induced whitespace cleanup) - see https://github.com/10up/classifai/pull/55/files?w=1 for the ignore whitespace changes view.

File: includes/Classifai/Command/ClassifaiCommand.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ClassifaiCommand extends \WP_CLI_Command {
 	 * ## Options
 	 *
 	 * [<post_ids>]
-	 * : Post IDs to classify
+	 * : Comma delimeted Post IDs to classify
```
Comment (helen) [inline includes/Classifai/Command/ClassifaiCommand.php]:
```suggestion
	 * : Comma-delimited list of post IDs to classify
```

**Pull request merged**